mariadb/repo.sls

```diff
@@ -1,3 +1,4 @@
+{%- set os_family = salt['grains.get']('os_family').lower() %}
```

Comment:
I believe this should work as well:

```
+{%- set os_family = salt['grains.get']('os_family')|lower %}

```
Jinja filters are chained using `|`, but perhaps the `.lower()` is more clear.